src/ansys/aedt/core/modeler/circuits/primitives_nexxim.py

```diff
@@ -850,6 +850,49 @@ def create_voltage_probe(self, name=None, location=None, angle=0, use_instance_i
         cmpid.set_property("Name", name)
         return cmpid
 
+    @pyaedt_function_handler(probe_name="name")
+    def create_current_probe(self, name=None, location=None, angle=0, use_instance_id_netlist=False):
+        """Create a current probe.
+
+        Parameters
+        ----------
+        name :
+            Name of the current probe. The default is ``None``.
```

Comment:
Not sure there is a default value when looking at the code below.
```suggestion
            Name of the current probe.
```

Reply:
There could be probes, similar to sources. It's better to keep them separately.

Reply:
@boyang2022 I'm not sure about which comment you are answering to. Could you explain this statement ?